packages/node/src/ethereum/api.ethereum.ts

```diff
@@ -115,32 +116,44 @@ export class EthereumApi implements ApiWrapper<EthereumBlockWrapper> {
     return this.client.getBlock(height);
   }
 
+  async getBlockPromise(num: number): Promise<any> {
+    const result = await retryOnFailEth(() =>
+      this.client.send('eth_getBlockByNumber', [hexValue(num), true]),
+    );
+    return result;
+  }
```

Comment:
```suggestion
  async getBlockPromise(num: number): Promise<any> {
    return retryOnFailEth(() =>
      this.client.send('eth_getBlockByNumber', [hexValue(num), true]),
    );
  }
```

Can be applied to tx receipt too